layouts/_default/baseof.html

```diff
@@ -1,4 +1,7 @@
 <!DOCTYPE html>
+{{ if .Site.Params.useTwemoji }}
```

Comment:
I think it's not the right place to insert this code, I would suggest placing it into the `<head>` element.